src/components/ClientCard/ClientCard.jsx

```diff
@@ -22,17 +26,20 @@ const ClientCard = props => (
 
 ClientCard.propTypes = {
   chat: PropTypes.object.isRequired,
+  config: PropTypes.object.isRequired,
   children: PropTypes.node.isRequired,
   setActiveChat: PropTypes.func.isRequired,
+  loadMessages: PropTypes.func.isRequired,
 };
 
 
-const mapStateToProps = (/* state */) => ({
-
+const mapStateToProps = state => ({
+  config: state.chat.config,
 });
 
 const mapDispatchToProps = dispatch => ({
   setActiveChat: chat => dispatch(setActiveChat(chat)),
+  loadMessages: (config, id) => loadMessages(dispatch, config, id),
```

Comment:
in Application.jsx you do the following:

```
 updateConfig: (event, config) => {
    // Set config
    dispatch(setConfig(config));

    // Then reload chats
    loadChats(dispatch, config);
  },
```

would you consider doing the same here? It seems that this component only handles one function that does both of these actions onClick.